src/index.html

```diff
@@ -20,7 +20,46 @@
     <meta property="fb:app_id" content="1618309211750238">
     <link rel="apple-touch-icon" sizes="180x180" href="touchicon.png">
 
-    <!--<link rel="apple-touch-startup-image" href="css/images/iossplash.png">-->
+    <!-------------------- iPhone 5 ------------------->
+    <link href="splashscreens/iphone5_splash.png" media="screen and (device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: portrait)" rel="apple-touch-startup-image" />
+    <link href="splashscreens/iphone5_splash_l.png" media="screen and (device-width: 320px) and (device-height: 568px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)" rel="apple-touch-startup-image" />
```

Comment:
It looks like the documentation [here](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html) doesn't require several images for different resolutions. We might also be able to just specify one image with a small icon to avoid all these splash screens, or maybe just three or four to use between different resolution screens.

Reply:
It used to be working with a single <link> entry but this is [no longer the case](https://medium.com/appscope/adding-custom-ios-splash-screens-to-your-progressive-web-app-41a9b18bdca3):

> What is not mentioned is that the href attribute has to refer to an image that is of the exact same resolution as the iOS device that uses the app. The problem that arises here is that there are multiple iOS devices with different resolutions, and unfortunately we cannot just simply repeat this code multiple times for images of different sizes. 

It's kind of ugly but this is the only way to do this currently. I'll try to merge them.

Reply:
If that's the case you can probably just leave them. You might be able to shorten the media attribute a bit by declaring the resolutions like `1080x1920` but that might not work with the orientation check.

Reply:
Resolution is not working with orientation indeed. This seems to be [the only working method](https://gist.github.com/EvanBacon/7fd4dc3be3d00096579bb0b134c56ec7) currently... Ii might get better with future releases of iOS. 

Android does not have this problem since the splash screen is made up from the manifest background color and the 512px icon for every device.